### PR TITLE
Fix float comparison error message

### PR DESCRIPTION
**Pull request by dbrgn**
I was at first a bit confused because I didn't find docs for `Float.within` :slightly_smiling_face:

### CODE REVIEW

File: src/Expect.elm [hunk 1]
```diff
@@ -313,6 +313,7 @@ minor inaccuracies introduced by floating point arithmetic.
     -- Fails because 0.1 + 0.2 == 0.30000000000000004 (0.1 is non-terminating in base 2)
     0.1 + 0.2 |> Expect.equal 0.3
 
+
```
Comment (harrysarson) [inline src/Expect.elm]: This extra line of white space is causing elm format to fail verification, if you remove it (or run elm-format) then CI should go green :)

Comment (dbrgn) [reply src/Expect.elm]: Oh sorry, I didn't notice that!

Comment (dbrgn) [reply src/Expect.elm]: Hm, actually this is the result of running `elm-format 0.19` on the file. Maybe you're still using 0.18 in CI?

Comment (harrysarson) [reply src/Expect.elm]: which version of elm-format are you running locally?

Comment (dbrgn) [reply src/Expect.elm]: elm-format-0.19 0.8.0. I saw that there's a patch release available, might be that this changed the behavior.

I simply remove the additional line. Should pass the tests now.

Comment (harrysarson) [reply src/Expect.elm]: Yeah, elm-format-0.19 0.8.1 is used in CI. Nice 1.

**Comment by mgold** [comment]
Thanks!

**Comment by mgold** [comment]
Oops, not docs, error messages.

**Pull request merged**